### DISCUSSION TITLE
【灌水】初到绳网，大家好

### DISCUSSION THREAD

**Discussion by Yeppisuo122** [share121/inter-knot#4956] [灌水]
话说今天更新的三个深渊水温怎样，有人打过了吗

**Comment by yihangyingzi** [comment]
星铁非常简单，原神没打，绝区零最后一层没打过

**Reply by HLXS-HLXS** [reply]
原神深渊考验良心，策划放了一大群小龙在那卖萌<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by Carkys** [comment]
绝区零第7层卡4:59过的s。本来打到最后时间刚超1s，给我变紫了，但出去结算还是s。

**Comment by linzhiyao123** [comment]
肥肠简单，全是宝宝巴士 :P
原神的怪像来卖萌的
崩铁还好，正常水温左右感觉
zzz之前二队没练上来没满，现在练上来了都60了打完余了1min（一队鲨鱼二队猫又派派）

**Comment by konglongyingzi** [comment]
还可以，zzz没打，其他几个也不难

**Comment by Zhangyifan2008** [comment]
我觉得崩铁和原都挺简单的，绝我只抽了鲨鱼妹和青衣常驻还没出C，出的柯蕾达和格莉丝
